src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -568,6 +568,12 @@ struct SvgNodeIdPair
     char *id;
 };
 
+struct EmbeddedFont
```

Comment:
font-face seems to allow src to contain a local path.
We support loading local ttf fonts. So it seems that font-face is not necessarily used only in the embedded font case.
Therefore, I think struct FontFace is more appropriate.

Reply:
done